spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -201,14 +208,15 @@ function onChanOpenAck(
   // port has already been validated
   // assert that version is "ics27-1"
   abortTransactionUnless(version === "ics27-1")
+  // state change to keep track of successfully registered interchain account
+  confirmInterchainAccountRegistration()
```

Comment:
This is where we would invoke a state change to keep a local registry of what accounts are registered on what chains. I'm wondering, can we pass back the address of the interchain account here? 

@AdityaSripal Is this what you had in mind?

Reply:
I like the idea of passing back account here

Reply:
1. No, channels are really not designed to be passing information. The only mechanism I can think of is using the channel version
2. Correct, although I'm a little confused. Isn't this the controlling side so won't the account be the portID? The counterparty can get this information if needed via the counterparty PortID
3. Packets cannot be received if the channel isn't OPEN. I don't see any reasons for concern. The handshake can always be completed unless there is a bug or the counterparty chain refuses to communicate. OnChanAck is like the account is created and OnChanConfirm is like the account is ready to be used